pkg/store/project/project.go

```diff
@@ -78,3 +78,17 @@ func (s *store) Exists(db *gorm.DB, id string) (bool, error) {
 	query := db.Raw("SELECT EXISTS (SELECT * FROM projects WHERE id = ?) as result", id)
 	return record.Result, query.Scan(&record).Error
 }
+
+// One get 1 project by id
+func (s *store) One(db *gorm.DB, id string) (*model.Project, error) {
+	var project *model.Project
+	return project, db.Where("id = ?", id).
+		Preload("Members", "deleted_at IS NULL and left_date IS NULL AND status IN ?",
+			[]model.AccountStatus{model.AccountStatusActive, model.AccountStatusOnBoarding}).
+		Preload("Members.Employee").
+		Preload("Heads", "deleted_at IS NULL and left_date IS NULL").
+		Preload("Heads.Employee").
+		Preload("Stacks").
```

Comment:
I'm not sure "Stacks" in project is the same with in employee. If it's similar, it should be check table project_stacks with condition deleted_at is null before preload stacks. @namnhce help me, please